test/WorkflowCore.Tests.SqlServer/SqlServerQueueProviderFixture.cs

```diff
@@ -0,0 +1,140 @@
+﻿#region using
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+using FluentAssertions;
+
+using WorkflowCore.Interface;
+using WorkflowCore.QueueProviders.SqlServer.Services;
+
+using Xunit;
+using Xunit.Abstractions;
+
+#endregion
+
+namespace WorkflowCore.Tests.SqlServer
+{
+    [Collection("SqlServerBroker collection")]
+    public class SqlServerQueueProviderFixture : IDisposable
+    {
+        #region Init
+
+        readonly SqlServerQueueProvider _qb;
+        private readonly ITestOutputHelper _console;
+
+        public SqlServerQueueProviderFixture(ITestOutputHelper output)
+        {
+            _console = output;
+            var connectionString = "Server=(local);Database=wfc;User Id=wfc;Password=wfc;";
+
+            _qb = new SqlServerQueueProvider(connectionString, "UnitTest", true);
+            _qb.Start().Wait();
+
+            while (_qb.DequeueWork(QueueType.Event, CancellationToken.None).Result != null) { }
+
+            while (_qb.DequeueWork(QueueType.Workflow, CancellationToken.None).Result != null) { }
+        }
+
+        public void Dispose()
+        {
+            _qb.Dispose();
+        }
+
+        #endregion
+
+        #region QueueDeque
+
+        [Fact]
+        public void QueueDequeTest()
```

Comment:
Could we use more descriptive test names?
Something that clearly indicates the requirement under test, formatted like `should_do_something`